app/src/main/res/layout/view_contributor_cell.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        >
+
+    <data>
+
+        <variable
+                name="viewModel"
+                type="io.github.droidkaigi.confsched2017.viewmodel.ContributorViewModel"
+                />
+    </data>
+
+    <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?attr/selectableItemBackground"
+            android:clickable="true"
+            android:onClick="@{viewModel::onClickContributor}"
+            >
+
+        <ImageView
+                android:id="@+id/avatar"
+                android:layout_width="@dimen/contributor_image"
+                android:layout_height="@dimen/contributor_image"
+                android:layout_centerHorizontal="true"
+                android:layout_margin="@dimen/contributor_cell_margin"
```

Comment:
> Do you mean to change only ImageView's margin bottom to 8dp while keeping margin top, left, right 16dp?

Yes, I want to change the margin between ImageView and TextView!

<img width="300" alt="implement_contributors_view_by_srym_ _pull_request__138_ _droidkaigi_conference-app-2017" src="https://cloud.githubusercontent.com/assets/1269214/22674961/242b484e-ed26-11e6-9197-4956c329233d.png">